src/Host/Client/Impl/Host/RHostConnector.cs

```diff
@@ -100,13 +106,12 @@ internal abstract class RHostConnector : IRHostConnector {
             }
         }
 
-        public async Task<RHost> ConnectAsync(string name, IRCallbacks callbacks, string rCommandLineArguments = null, int timeout = 3000, CancellationToken cancellationToken = new CancellationToken()) {
+        public async Task<RHost> ConnectAsync(string name, IRCallbacks callbacks, int timeout = 3000, CancellationToken cancellationToken = new CancellationToken()) {
```

Comment:
Good point. I'll revert this

---
In reply to: [75950345](https://github.com/Microsoft/RTVS/pull/2192#discussion_r75950345) [](ancestors = 75950345,75949601,75946453)